images/iqsharp-base/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# The same image used by mybinder.org
+0.22.192090-beta0.22.192090-beta# The same image used by mybinder.org
```

Comment:
This seems like a mistaken replacement... I've seen this happen if the global find-replace is used in the GitHub web editor. It adds the replacement text to the top of the file instead of the right location in the middle of the file. This appears to have happened to a few of the files in here. Unfortunately that means this change will have to be done locally with global find-replace or manually in the web editor.

Reply:
Absolutely is, thank you for catching that! Let me go fix that, then.